include/hipipe/core/python/utility/pyboost_cv_point_converter.hpp

```diff
@@ -0,0 +1,43 @@
+/****************************************************************************
+ *  hipipe library
+ *  Copyright (c) 2018, Iterait a.s.
+ *  Author(s) Adam Blazek
+ *
+ *  This file is distributed under the MIT License.
+ *  See the accompanying file LICENSE.txt for the complete license agreement.
+ ****************************************************************************/
+
+#pragma once
+#if defined HIPIPE_BUILD_PYTHON && defined HIPIPE_BUILD_PYTHON_OPENCV
+
+// this header has to be included before the numpy header
+#include <hipipe/core/python/initialize.hpp>
+
+#include <Python.h>
+#include <boost/python.hpp>
+#include <numpy/ndarrayobject.h>
+#include <opencv2/core/core.hpp>
+
+#include <cstdio>
+
+namespace hipipe::python::utility {
+
+struct pointToTupleBoostConverter {
+    static PyObject* convert(cv::Point2f const& pt);
```

Comment:
Just for the sake of consistency:
```suggestion
    static PyObject* convert(const cv::Point2f& pt);
```

Reply:
https://github.com/iterait/hipipe/pull/20/files/2dc1727b208bee69109ea91a6676559f12806ec1#diff-ed2ec7969655b49827bb7fb72f6f2bd3L33

Reply:
Third party code. :-P